### PR TITLE
fix: fix package location of visibility conditions

### PR DESCRIPTION
**Pull request by dgriffon**
Fixes https://github.com/Jahia/jahia-ee/issues/1356

### Description
Since https://github.com/Jahia/jcontent/pull/2287 being merged the following exception show up in the logs when visibility conditions were set:
```
org.jahia.ajax.gwt.client.service.content.JahiaContentManagementService.getNodesAndTypes(java.util.List,java.util.List) throws org.jahia.ajax.gwt.client.service.GWTJahiaServiceException. Cause: Cannot invoke "org.jahia.data.templates.JahiaTemplatesPackage.getResourceBundleHierarchy()" because "pkg" is null

java.lang.NullPointerException: Cannot invoke "org.jahia.data.templates.JahiaTemplatesPackage.getResourceBundleHierarchy()" because "pkg" is null

	at org.jahia.utils.i18n.ResourceBundles.get(ResourceBundles.java:100) ~[jahia-impl-8.2.4.0-SNAPSHOT.jar:?]

	at org.jahia.utils.i18n.Messages.get(Messages.java:111) ~[jahia-impl-8.2.4.0-SNAPSHOT.jar:?]

	at org.jahia.modules.visibility.conditions.StartEndDateConditionRuleImpl.getGWTDisplayTemplate(StartEndDateConditionRuleImpl.java:37) ~[?:?]

	at org.jahia.ajax.gwt.helper.NodeHelper.populateVisibilityInfo(NodeHelper.java:1049) ~[jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	at org.jahia.ajax.gwt.helper.NodeHelper.getGWTJahiaNode(NodeHelper.java:218) ~[jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	at org.jahia.ajax.gwt.helper.NavigationHelper.getGWTJahiaNode(NavigationHelper.java:746) ~[jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	at org.jahia.ajax.gwt.content.server.JahiaContentManagementServiceImpl.getNodesInternal(JahiaContentManagementServiceImpl.java:418) ~[jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	at org.jahia.ajax.gwt.content.server.JahiaContentManagementServiceImpl.getNodesAndTypes(JahiaContentManagementServiceImpl.java:446) ~[jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	... suppressed 2 lines

	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]

	at org.jahia.ajax.gwt.commons.server.JahiaRPC.invokeAndEncodeResponse(JahiaRPC.java:84) [jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	at org.jahia.ajax.gwt.commons.server.JahiaRPC.invokeAndEncodeResponse(JahiaRPC.java:69) [jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	at org.jahia.ajax.gwt.commons.server.GWTController.processCall(GWTController.java:264) [jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373) [gwt-servlet-2.8.2.jar:?]

	at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62) [gwt-servlet-2.8.2.jar:?]

	at org.jahia.ajax.gwt.commons.server.GWTController.handleRequest(GWTController.java:147) [jahia-gwt-8.2.4.0-SNAPSHOT.jar:?]

	... suppressed 13 lines
```

This happen because a remaining reference to the `Jahia Visibitly` remained in the label lookup

That made the import test in jahia-ee to fail (see https://github.com/Jahia/jahia-ee/issues/1356)

### CODE REVIEW

**Comment by gflores-jahia** [comment]
Thank you @dgriffon, but looks like Cedric also has a similar PR https://github.com/Jahia/jcontent/pull/2371.

Closing this one as duplicate.
